tasks/agent.py

```diff
@@ -152,16 +152,28 @@ def build(ctx, rebuild=False, race=False, build_include=None, build_exclude=None
     }
     ctx.run(cmd.format(**args), env=env)
 
-    # Render the configuration file template
-    #
-    # We need to remove cross compiling bits if any because go generate must
-    # build and execute in the native platform
+    # Remove cross-compiling bits to render config
     env.update({
         "GOOS": "",
         "GOARCH": "",
     })
-    cmd = "go generate {}/cmd/agent"
-    ctx.run(cmd.format(REPO_PATH), env=env)
+
+    # Render the Agent configuration file template
+    cmd = "go run {go_file} {build_type} {template_file} {output_file}"
+
+    args = {
+        "go_file": "./pkg/config/render_config.go",
+        "build_type": "agent-py2py3" if with_both_python else "agent-py3",
```

Comment:
Is `agent-py3` also used for when there is only python2? If it's the case, maybe find a better name?